### PR TITLE
add trial-by-trial option for success metrics

### PR DESCRIPTION
**Pull request by leoscholl**

### CODE REVIEW

File: aopy/analysis/behavior.py [hunk 1]
```diff
@@ -15,67 +15,120 @@ def calc_success_percent(events, start_events=[b"TARGET_ON"], end_events=[b"REWA
...
+def calc_success_rate_trials(trial_success, trial_time, window_size=None):
+    '''
+    Calculate the number of successful trials per second with a given trial start and end definition. 
+
+    See also:
+        :func:`~aopy.analysis.calc_success_rate`
+
+    Args:
+        trial_success ((ntrial,) bool array): list of trials where success is non-zero and failure is zero
+        trial_time ((ntrial,) array): list of the time taken in each time (e.g. acquisition time)
```
Comment (rcanfield2) [inline aopy/analysis/behavior.py:109]: time taken in each trial

**Review by rcanfield2 — COMMENTED** [review]
Tests?

**Comment by leoscholl** [comment]
added tests in the same place as the tests for the events/event_times versions of the functions. i don't think comprehensive tests are required since the new functions get called by the old function anyway.

**Comment by leoscholl** [comment]
replaced by #351

**Pull request closed without merging**